build.sh

```diff
@@ -92,6 +93,16 @@ cd /tmp/local-repo
 makepkg --printsrcinfo >.SRCINFO
 echo '::endgroup::'
 
+if [ -n "$run_command" ]; then
+	echo '::group::Installing package with makepkg and run command'
+  git clone https://aur.archlinux.org/yay-bin.git && cd yay-bin && makepkg -si --noconfirm
+	cd /tmp/local-repo/
+  grep -E 'depends' .SRCINFO | cut -f 3 -d ' '| sed -e 's/://' | xargs yay -S --noconfirm
```

Comment:
Why is `yay` necessary? Why can't you execute `makepkg -si` on the current `PKGBUILD` directly?